requirements.txt

```diff
@@ -1,5 +1,5 @@
 jedi-language-server==0.41.1
 pytest==7.3.1
-pydantic>=1.10.5, <2
 pytest-asyncio==0.21.1
 requests==2.32.3
+typing-extensions==4.12.2
```

Comment:
Can you change this to typing-extensions>=4.2.0 as per (https://github.com/pydantic/pydantic/blob/0cbba0f9774c470e8d8a2a62abb921b8cea4027d/pyproject.toml#L39)